coreos-base/flatcar-eks/files/download-kubelet.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+# Get the cluster name from /etc/eks/cluster.conf
+. /etc/eks/cluster.conf
+if [[ -x ${CLUSTER_NAME} ]]; then
```

Comment:
Cluster name is supposed to be an executable or did you mean using `-z`?

Reply:
Yeah, it was supposed to be -z, thanks for catching this. I don't know how it didn't fail in my tests.